packages/react/examples/codesandbox/patterns/blocks/ContentBlockSegmented/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "parcel-react-ibmdotcom",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "@carbon/ibmdotcom-react": "1.7.0",
+    "@carbon/ibmdotcom-styles": "1.7.0",
```

@RaphaelAmadeu Let's use `latest` here?

```suggestion
    "@carbon/ibmdotcom-react": "latest",
    "@carbon/ibmdotcom-styles": "latest",
```